workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/compute.py

```diff
@@ -229,16 +227,6 @@ def _assign_diagnostic_time_attrs(
     return diagnostics_ds
 
 
-def dump_nc(ds: xr.Dataset, f):
```

Comment:
We switched to using the `vcm.dump_nc` version of this a while ago, so this func is unused.